packages/fxa-auth-server/test/local/routes/subscriptions/stripe-webhooks.js

```diff
@@ -345,6 +345,11 @@ describe('StripeWebhookHandler', () => {
           commonIgnorableErrorTest(error.emailBouncedHard(42))
         );
 
+        it(
+          'ignores emailComplaint',
+          commonIgnorableErrorTest(error.emailComplaint(42))
```

Comment:
Not a blocker for this patch, but I think there may be an issue with our error filtering here.

I noticed some of our webhook handlers have their own `try..catch` block (e.g. [`handleInvoicePaidEvent`](https://github.com/mozilla/fxa/blob/7078624f185784f0cbfacfcdfe84660bd09a3121/packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts#L399-L402)), and some, like the [`handleCustomerSourceExpiringEvent`](https://github.com/mozilla/fxa/blob/7078624f185784f0cbfacfcdfe84660bd09a3121/packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts#L427-L436) handler used in this test, do not.

In the `handleInvoicePaidEvent` case, it does not rethrow after catching the error. Wouldn't we only pass errors through this `IGNORABLE_STRIPE_WEBHOOK_ERRNOS` filter in the cases that the handlers either catch and rethrow or have no `try...catch` at all? I'm concerned that legitimate errors that we should not ignore are not being rethrown in some cases, as in `handleInvoicePaidEvent`.

Reply:
Good catch, and generally I agree.  But without looking through the code I can't be sure what sort of error that catch in `handleInvoicePaidEvent` is catching; there's no guarantee that it's an error from lib/error.js of auth-server, where those `IGNORABLE_STRIPE_WEBHOOK_ERRNOS` originate.  Do you want to file a "maintenance" issue so we can looking into it at some point?

Reply:
Sure; I filed https://mozilla-hub.atlassian.net/browse/FXA-3963.